test/e2e/framework/container.go

```diff
@@ -24,18 +24,16 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/util/uuid"
 	podutil "k8s.io/kubernetes/pkg/api/v1/pod"
-	"k8s.io/kubernetes/test/e2e/framework"
 )
 
 // One pod one container
-// TODO: This should be migrated to the e2e framework.
```

Comment:
@deads2k I notice a todo here, do I misunderstand it?

Reply:
> @deads2k I notice a todo here, do I misunderstand it?

I think the todo is out of date.  How about removing the todo, leaving the code where it is, and merging that instead?

Reply:
OK, I done it , Thanks.